hydra-node/src/Hydra/Chain/Direct/Handlers.hs

```diff
@@ -133,6 +133,14 @@ finalizeTx TinyWallet{sign, coverFee} ctx ChainStateAt{chainState} partialTx = d
       throwIO (NoFuelUTXOFound :: PostTxError Tx)
     Left ErrNotEnoughFunds{} ->
       throwIO (NotEnoughFuel :: PostTxError Tx)
+    Left ErrScriptExecutionFailed{} ->
+      throwIO
+        ( PlutusValidationFailed
+            { plutusFailure = "" -- TODO: extract this
+            , plutusDebugInfo = "" -- TODO: extract this or keep empty or defina new error type
```

Comment:
Must: resolve these TODOs

Reply:
`plutusFailure` has been extracted but `plutusDebugInfo` has been left empty, is it ok?

Reply:
I'm going to resolve this comment as it is being continued [below](https://github.com/input-output-hk/hydra/pull/680#discussion_r1096272714).